include/yaramod/parser/location.h

```diff
@@ -21,13 +22,19 @@ class Location
 	 */
 	struct Position {
 
+		Position() : line(1), column(0) {}
```

Comment:
I think it would be beneficial to add comment about how we count lines and columns.

- lines from 1
- columns from 0